src/main/java/kg/jarkyn/server/Request/ParsedRequest.java

```diff
@@ -0,0 +1,19 @@
+package kg.jarkyn.server.Request;
+
+public class ParsedRequest {
+    private final String verb;
+    private final String path;
+
+    public ParsedRequest(String verb, String path) {
```

Comment:
Could the 'verb' be limited to an Enum? We know which verbs are allowed and doing this as an Enum will increase the type safety...